README.md

```diff
@@ -2,42 +2,17 @@
 
 # Ember-Electron
 
-![Latest release on NPM](https://img.shields.io/npm/v/ember-electron.svg) [![Build status](https://ci.appveyor.com/api/projects/status/5rhwhar361uad07v?svg=true)](https://ci.appveyor.com/project/adopted-ember-addons/ember-electron)
- ![Travis CI Build Status](https://secure.travis-ci.org/adopted-ember-addons/ember-electron.svg?branch=master) ![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-electron.svg)
+![Latest release on NPM](https://img.shields.io/npm/v/ember-electron.svg) 
+[![Build status](https://ci.appveyor.com/api/projects/status/5rhwhar361uad07v?svg=true)](https://ci.appveyor.com/project/adopted-ember-addons/ember-electron)
```

Comment:
Let's specify `Appveyor` here?

Reply:
Do what? I'm not familiar with Appveyor.

Reply:
Appveyor is a CI system like travis, but for running windows builds. This link points to the appveyor build status, but the text of the link it `Build status`, so I'm just proposing we change it to say `Appveyor Build Status` (like we have `Travis Build Status` as the text of the travis link).